db/migrate/20250515081023_add_connected_apps_table.rb

```diff
@@ -0,0 +1,5 @@
+class AddConnectedAppsTable < ActiveRecord::Migration[7.1]
```

Comment:
Can we change the name of this and the file name too, since we're not adding a new table anymore?

Reply:
thanks for spotting this! 🤦‍♂️